line-bot-spring-boot/build.gradle

```diff
@@ -14,12 +14,15 @@
  * under the License.
  */
 
+apply plugin: 'java-library'
+
 dependencies {
-    compile project(':line-bot-api-client')
-    compile project(':line-bot-model')
-    compile project(':line-bot-servlet')
-    compile 'org.springframework.boot:spring-boot-autoconfigure'
-    compile 'org.springframework.boot:spring-boot-starter-web'
+    api project(':line-bot-api-client')
+    implementation project(':line-bot-model')
+    implementation project(':line-bot-servlet')
+    implementation 'org.springframework.boot:spring-boot-autoconfigure'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
`line-bot-spring-boot` uses `org.springframework.boot:spring-boot-starter-web` as it's api, not internal implementation only.
So use `api` here.

Reply:
Yep... But every line-bot-spring-boot users should depend on `spring-boot-starter-web`.
As a result, spring-boot-starter-web should be `implementation` here.

Reply:
ok